src/edgepi/dac/edgepi_dac.py

```diff
@@ -50,26 +65,43 @@ def __init__(self):
         self.gpio = EdgePiGPIO()
 
         self.__dac_power_state = {
+            DACChannel.AOUT8.value: PowerMode.NORMAL.value,
             DACChannel.AOUT7.value: PowerMode.NORMAL.value,
             DACChannel.AOUT6.value: PowerMode.NORMAL.value,
             DACChannel.AOUT5.value: PowerMode.NORMAL.value,
             DACChannel.AOUT4.value: PowerMode.NORMAL.value,
             DACChannel.AOUT3.value: PowerMode.NORMAL.value,
             DACChannel.AOUT2.value: PowerMode.NORMAL.value,
             DACChannel.AOUT1.value: PowerMode.NORMAL.value,
```

Comment:
Is there a reason why this is in descending order whereas mostly everywhere else it's in ascending order?

Reply:
I don't think there was a reason for the order.